src/azure-cli-core/azure/cli/core/profiles/_shared.py

```diff
@@ -245,7 +266,7 @@ def get_versioned_sdk_path(api_profile, resource_type, operation_group=None):
                       azure.mgmt.storage.v2016_12_01.operations.storage_accounts_operations
     """
     api_version = get_api_version(api_profile, resource_type)
-    if isinstance(api_version, tuple):
+    if isinstance(api_version, _ApiVersions):
```

Comment:
There is more tests like this:
- In `src/azure-cli-core/azure/cli/core/commands/__init__.py` (three when I did the PR):
  - register_cli_argument
  - register_extra_cli_argument
  - cli_command

https://github.com/Azure/azure-cli/pull/4983

Reply:
That was the dev branch (before knack). Many of these methods are replaced in knack branch.

Reply:
If you say so :)